dstep/main.d

```diff
@@ -56,19 +56,10 @@ auto parseCLI (string[] args)
         args,
         std.getopt.config.passThrough,
         std.getopt.config.caseSensitive,
-        "version", &config.dstepVersion,
-        "clang-version", &config.clangVersion,
-        "output|o", "Write output to", &config.output,
-        "language|x", "Treat subsequent input files as having type <language>.", &parseLanguage,
-        "objective-c", "Treat source input file as Objective-C input.", &forceObjectiveC,
-        "no-comments", "Disable translation of comments.", &config.noComments,
-        "public-submodules", "Use public imports for submodules.", &config.publicSubmodules,
-        "package", "Specify package name.", &config.packageName,
-        "dont-reduce-aliases", "Disable reduction of primitive type aliases.", &config.dontReduceAliases,
-        "no-portable-wchar_t", "Translate wchar_t to wchar or dchar depending on its size.", &config.noPortableWCharT,
-        "single-line-function-headers", "Do not break function headers to multiple lines.", &config.singleLineFunctionHeaders,
-        "no-space-after-function-name", "Do not put a space after a function name.", &config.noSpaceAfterFunctionName,
-        "zero-param-is-vararg", "Translate functions with empty argument list as variadic functions.", &config.zeroParamIsVararg);
+        "output|o", &config.output,
+        "language|x", &parseLanguage,
+        "objective-c", &forceObjectiveC,
```

Comment:
I would like to keep the `objective-c` flag. It allows to have a consistent looks of the flags. `-ObjC` is not consistent but I added that because that's flag is available in Clang.